packages/govuk-frontend-review/src/views/full-page-examples/travel-guidance/index.njk

```diff
@@ -0,0 +1,396 @@
+    
+    {{ govukPagination({
+       previous: { href: '#', labelText: 'Safety and security' },
+       next: { href: '#', labelText: 'Time dilation' }
+    }) }}
+  </div>
+</div>
+{% endblock %}
+
+{% block footer %}
+
+{{ govukFooter({
+  navigation: [
+    {
+      title: "Services and information",
+      width: "two-thirds",
+      columns: 2,
+      items: [
+        {
+          href: "#",
+          text: "Benefits"
+        },
+        {
+          href: "#",
+          text: "Births, deaths, marriages and care"
+        },
+        {
+          href: "#",
+          text: "Business and self-employed"
+        },
+        {
+          href: "#",
+          text: "Childcare and parenting"
+        },
+        {
+          href: "#",
+          text: "Citizenship and living in the UK"
+        },
+        {
+          href: "#",
+          text: "Crime, justice and the law"
+        },
+        {
+          href: "#",
+          text: "Disabled people"
+        },
+        {
+          href: "#",
+          text: "Driving and transport"
+        },
+        {
+          href: "#",
+          text: "Education and learning"
+        },
+        {
+          href: "#",
+          text: "Employing people"
+        },
+        {
+          href: "#",
+          text: "Environment and countryside"
+        },
+        {
+          href: "#",
+          text: "Housing and local services"
+        },
+        {
+          href: "#",
+          text: "Money and tax"
+        },
+        {
+          href: "#",
+          text: "Passports, travel and living abroad"
+        },
+        {
+          href: "#",
+          text: "Visas and immigration"
+        },
+        {
+          href: "#",
+          text: "Working, jobs and pensions"
+        }
+      ]
+    },
+    {
+      title: "Departments and policy",
+      width: "one-third",
+      items: [
+        {
+          href: "#",
+          text: "How government works"
+        },
+        {
+          href: "#",
+          text: "Departments"
+        },
+        {
+          href: "#",
+          text: "Worldwide"
+        },
+        {
+          href: "#",
+          text: "Policies"
+        },
+        {
+          href: "#",
+          text: "Publications"
+        },
+        {
+          href: "#",
+          text: "Announcements"
+        }
+      ]
+    }
+  ],
+  meta: {
+    items: [
+      {
+        href: "#",
+        text: "Help"
+      },
+      {
+        href: "#",
+        text: "Cookies"
+      },
+      {
+        href: "#",
+        text: "Contact"
+      },
+      {
+        href: "#",
+        text: "Terms and conditions"
+      },
+      {
+        href: "#",
+        text: "Rhestr o Wasanaethau Cymraeg"
+      }
+    ],
+    html: 'Built by the <a href="#" class="govuk-footer__link">Government Digital Service</a>'
+  }
+}) }}
+
+{% endblock %}
+
+{% block styles %}
+<style>
+/* Styles for the GOV.UK Publishing Components contents list. */
```

Comment:
Mind if we move these styles into [`./stylesheets/full-page-examples`](https://github.com/alphagov/govuk-frontend/tree/main/packages/govuk-frontend-review/src/stylesheets/full-page-examples) as `travel-guidance.scss`?

Reply:
Trying to pick a consistent approach we're all happy with is more important

Typically it'd be to tap into our setup so it gets free upgrades as we go:

1. Stylelint for code quality checks
2. Sass tools, helpers (colours, font stacks, type scale, breakpoints)
3. PostCSS transforms (Autoprefixer, CSSNANO)

But if the lift-and-shift examples should have a separate `<style>` tag I'm happy with that

I don't know how far we typically go to adhere them to GOV.UK Frontend, avoiding tech debt

Reply:
Just bringing over a conversation from Slack

Accepting that this is tech debt that we'll need to support during the new type scale work

But in reply from @36degrees:

>This has been at the back of my mind for a while. I think we want the full-page examples as ‘realistic’ as possible but ideally use as little extra CSS as possible. I don’t think we need to precisely recreate the layout as it is on the pages we’re mimicking, just get close enough that e.g. DAC can understand what it is they’re looking at.
>
>I think on that basis my preference for the new travel advice page would be to just use a standard H2 and a bulleted list. I would quite like to at some point simplify the search example. The campaign page may be trickier, but also I’m not sure how useful it is any more.
>
>I don’t think we need to eradicate extra CSS entirely, but I think we should try and keep it to a minimum. Inevitably they diverge from the original design over time anyway.
Loosely held opinion, so definitely open to hearing alternative views.

Do we need to tweak anything @querkmachine?

Reply:
It sounds like the intent going forward is to try and be 'similar but not the same' with regard to non-Frontend components in examples, so I can strip this down to a basic heading and list, sure.